actions/raw_action.go

```diff
@@ -12,7 +12,12 @@ Yaml syntax:
 
 Mandatory properties:
 
-- origin -- reference to named file or directory.
+- origin -- reference to directory or previous action. Possible values:
+
+  1) "recipe" ....... directory the recipe is in
```

Comment:
this documentation should be moved somewhere generic, since `recipe` is generic we only want to update it once.